plugins/lcvcore/src/qgradient.cpp

```diff
@@ -0,0 +1,124 @@
+#include "qgradient.h"
+#include <QVector3D>
+#include "qmat.h"
+#include "opencv2/core.hpp"
+#include <opencv2/core/matx.hpp>
+
+#define GAMMA 0.43

```

Comment:
Should be a const of QGradient.

class QGradient{ const double Gamma = 0.43; }